backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -376,8 +376,8 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
   }
 
   test("test 'function rand'") {
-    runSql("select rand(), rand(1), rand(null) from range(10)")(
-      checkOperatorMatch[ProjectExecTransformer])
+    // TODO: add rand(1), rand(null) test back.
+    runSql("select rand() from range(10)")(checkOperatorMatch[ProjectExecTransformer])
```

Comment:
Hi @taiyang-li, this PR removes CH's testing for `rand(seed)`. This PR make such case fall back. See PR description. Please let me know if you have other ideas. Thanks!

Reply:
cc @zzcclp

Reply:
@PHILO-HE Maybe we can add a if in RandTransformer for clickhouse backend to make sure CH do not fallback.

Reply:
> @PHILO-HE Maybe we can add a if in RandTransformer for clickhouse backend to make sure CH do not fallback.

Hi @taiyang-li, does CH already handle the issue as mentioned in description? I'm assuming CH has the same issue which needs a separate fix, so I did this change.

Reply:
Thanks for reminding. Currently CH doesn't need the feature you mentioned in PR. Because CH rand function doesn't support user speicified seed.